recipes/nsync/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.23.0":
```

Comment:
There is newer version available. Is there a reason to add older one?

Reply:
@Nekto89 right, but 1.7.1 `onnxruntime` depends on this one, this is why I decided to add this one